WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -45,47 +45,49 @@ final class PaymentCaptureOrchestrator {
 
         ServiceLocator.stores.dispatch(setAccount)
 
-        guard let parameters = paymentParameters(
-                order: order,
-                statementDescriptor: paymentGatewayAccount.statementDescriptor,
-                paymentMethodTypes: paymentMethodTypes
-        ) else {
-            DDLogError("Error: failed to create payment parameters for an order")
-            onCompletion(.failure(CardReaderServiceError.paymentCapture()))
-            return
-        }
-
-        /// Briefly suppress pass (wallet) presentation so that the merchant doesn't attempt to pay for the buyer's order when the
-        /// reader begins to collect payment.
-        ///
-        suppressPassPresentation()
+        Task {
+            do {
+                let parameters = try await paymentParameters(
```

Comment:
I don't think we can use `async await` yet because we still support iOS 14... or am I out of the loop here?

Reply:
It gained backward compatibility as you can see [here](https://www.swiftbysundell.com/special/swift-concurrency-backward-compatibility/), but depending on the decision we take [here](https://github.com/woocommerce/woocommerce-ios/pull/6585#discussion_r844910161) I can remove it because it wouldn't be (that) necessary

Reply:
this looks nice! my understanding of using async/await is that we have to wait for CI to upgrade the Xcode version to at least 13.2. Right now it's on 13.0:

https://github.com/woocommerce/woocommerce-ios/blob/6242f86223d2329d45a8b6702e6b443d9622005b/.circleci/config.yml#L18

Reply:
Thanks, @jaclync for pointing that out! In that case, we cannot use it, because Xcode 13.0 only allows `async/await` for iOS 15. I removed it, a pity 😅